### ISSUE TITLE
Does exist any solution to backup all configs?

### ISSUE THREAD

**Issue by phelipemaia** [OptimalBits/node_acl#122]
Hi guys,

My concern is about the integrity of my configuration. Node_acl will store everything in a "non-trustable" database. So, I would like to know if there are any solution to create a backup or a restore easily.

Thanks

**Comment by phelipemaia** [comment]
Let me be clear about "non-trustable": if I restart my redis I will lost every data. I know that I can use some strategies on redis like RDB or AOF, but what is the best solution? Is to keep all config in other database to recreate all redis structure after a restart, invoking the functions to set permissions?

**Comment by jbeckton** [comment]
Maybe you can keep your configs in js code as part of your app and when it starts up it can seed the redis storage? Similar to MongoDB dbconnection.once() method that is run when the app starts and the connection is established I have some roles and perms created in the db.

**Comment by phelipemaia** [comment]
@jbeckton Yep but what I tried to explain is that permissions has no backups. Once I restart my redis I lose every permission created before.
